### ISSUE TITLE
[Bug] [CameraView] [Android] Taken photos are saved but don't appear in the gallery.

### ISSUE THREAD

**Issue by acuntex** [xamarin/XamarinCommunityToolkit#298]
### Description
When taking a photo with CameraView and SavePhotoToFile is true, the photo gets saved to the storage but does not appear in the gallery.
I've also tried to use Google Photos and the OneDrive app, but none of them detect the image.

### Steps to Reproduce

1. Create CameraView with `SavePhotoToFile="true"`
2. Take a photo

(this also can be reproduced in the sample app)

### Expected Behavior
Photo apps that usually show saved photos should show the taken photo.

### Actual Behavior
Photo apps that usually show saved photos do not show the taken photo.

**Comment by bmacombe** [comment]
I'm not sure that feature was meant to add them to photo gallery.  Right now it uses a hard coded path on each platform but the original design used platform specifics to let that be set.  Once we figure out how to get platform specifics working, they are intneral to XF, we can add that back.

**Comment by jfversluis** [comment]
Agreed with @bmacombe although I would find it more logical to save an image to the default gallery than some random path. I think that might be a separate new property to arrange that though and either remove this property or make if more configurable by allowing users to set a path of sorts

**Comment by bmacombe** [comment]
I'm good with adding to the gallery, as long as that's a configurable option.

**Comment by jfversluis** [comment]
I'm a big fan of making as much configurable as possible. So definitely agree!
